sdl_android_lib/src/com/smartdevicelink/transport/SdlBroadcastReceiver.java

```diff
@@ -203,7 +203,12 @@ private static boolean isRouterServiceRunning(Context context, boolean pingServi
 	            	Intent intent = new Intent();
 	            	intent.setClassName(service.service.getPackageName(), service.service.getClassName());
 	            	intent.putExtra(TransportConstants.PING_ROUTER_SERVICE_EXTRA, pingService);
-	            	context.startService(intent);
+	            	try {
+				context.startService(intent);
+			}catch (SecurityException e){
+				Log.e(TAG, "Process is bad");
+				android.os.Process.killProcess(android.os.Process.myPid()); // Let's exit, we can't start our service
```

Comment:
We don't want to do this. Killing myPid process will actually kill the entire app. We just want to fail silently here, add a return statement there instead.